.github/workflows/continuous-integration.yml

```diff
@@ -466,26 +466,19 @@ jobs:
         env:
           YARN_CACHE_FOLDER: .cache/yarn
 
-      - name: Run git diff and set DIFF_RESULTS
-        if: ${{ github.ref != 'refs/heads/master' }}
-        id: git-diff
-        run: echo ::set-output name=DIFF_RESULTS::$(git diff ${{ github.event.before }} ${{ github.event.after }} --line-prefix=new_line --unified=0)
-
-      - name: Set IS_GRAPH_UPDATED and update the cross-app import graph if necessary
+      - name: Regenerate the cross-app import graph
         if: ${{ github.ref != 'refs/heads/master' }}
         run: node script/github-actions/create-cross-app-import-graph.js
-        env:
-          DIFF_RESULTS: ${{ steps.git-diff.outputs.DIFF_RESULTS }}
 
       - name: Push the updated cross-app import graph
-        if: ${{ github.ref != 'refs/heads/master' && env.IS_GRAPH_UPDATED == 'true' }}
+        if: ${{ github.ref != 'refs/heads/master' }}
```

Comment:
Do we need a condition to abort the rest of the build if there's an update to the graph? The commit with the graph update will re-trigger the build, so there's no point in running the rest of the workflow the first time this runs.

Reply:
Good idea Darius. Let me figure out a way to set that conditional.

Reply:
This is working now.

The `stefanzweifel/git-auto-commit-action@v4` action provides the following output `steps.commit.outputs.changes_detected ` (when you give the step it's used an `id`, like `commit` in this case) which we can use to cancel the current workflow when `steps.commit.outputs.changes_detected  == 'true'` (when the graph was updated and pushed).

The resulting commit will not trigger another GitHub Actions Workflow run. This is due to limitations set by GitHub.

> When you use the repository's GITHUB_TOKEN to perform tasks on behalf of the GitHub Actions app, events triggered by the GITHUB_TOKEN will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs.

This can be overridden by passing `VA_VSP_BOT_GITHUB_TOKEN` to the actions/checkout Action step.